fhir-core/src/main/java/com/ibm/fhir/core/FHIRUtilities.java

```diff
@@ -117,7 +118,7 @@ public static boolean isEncoded(String s) {
      * @return
      */
     public static Timestamp convertToTimestamp(java.time.ZonedDateTime zdt) {
-        return new Timestamp(zdt.toInstant().toEpochMilli());
+        return Timestamp.from(zdt.withZoneSameInstant(ZoneId.of("UTC")).toInstant());
```

Comment:
This one still confuses me.  An Instant doesn't have a dateTime, so why do we need to convert the ZonedDateTime to UTC before calling toInstant?

Reply:
this step ensures it is UTC.  when it's converted to an instant

Reply:
Instant = 1576849840146
Converted to Timestamp
Fri Dec 20 2019 13:51:08
however that instant is actually the time EST
so we need to adjust UP
Instant = 1576859840146
Fri Dec 20 2019 16:37:20
Now, it's actually in UTC (edited)